loleaflet/src/control/Control.UserList.js

```diff
@@ -101,7 +102,27 @@ L.Control.UserList = L.Control.extend({
 		return content;
 	},
 
-	updateUserListCount: function() {
+	renderUserAvatars: function(viewId, userName, extraInfo, color) {
+		if (extraInfo !== undefined && extraInfo.avatar !== undefined) {
```

Comment:
This check is duplicate with the one in https://github.com/CollaboraOnline/online/pull/3529/files#diff-41de9cb6f19860d72e4fe0ba9bae4682ea79c3312a3ca4ad3c1219cf43f0d8efR88